### PR TITLE
Update pyglet description in the libraries page. 

### PR DESCRIPTION
**Pull request by benmoran56**
I noticed that most descriptions are quite short. Let me know if it's too wordy, and I'll see if I can shorten it a bit.

### CODE REVIEW

File: docs/libraries.rst [hunk 1]
```diff
@@ -51,13 +51,14 @@ loop and tweening.
 :Website: http://pyglet.org/
 :Documentation: https://pyglet.readthedocs.io/
 
-Pyglet is a game framework that offers functionality for 2D graphics and text
-rendering using OpenGL. It also has complete OpenGL bindings, and can be used
-as the basis of 3D games.
+pyglet provides hardware accelerated rendering for sprites, drawing text,
+audio playback, and joystick support. Sprites can be static or animated, 
+rotated, scaled, and support transparency. pyglet is build on OpenGL,
```
Comment (lordmauve) [inline docs/libraries.rst]: Typo, "built"?

**Comment by benmoran56** [comment]
Oops, I made a quick commit to fix that.
